src/lib/rollup.ts

```diff
@@ -1,20 +1,18 @@
 import alias from '@rollup/plugin-alias';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
 import type { Plugin, RollupOutput } from 'rollup';
 import { rollup } from 'rollup';
+import css from 'rollup-plugin-css-only';
 import svelte from 'rollup-plugin-svelte';
 import { terser } from 'rollup-plugin-terser';
 import preprocess from 'svelte-preprocess';
-import typescript from '@rollup/plugin-typescript';
-import css from 'rollup-plugin-css-only';
 import type { Props } from './svelte';
 
 const performancescript = `window.performance.mark('svelteEnd');
 const measure = window.performance.measure('svelte', 'svelteStart','svelteEnd');
 document.querySelector("#metrics").innerText = \`\${measure.duration.toFixed(2)}ms\`;`;
 
-const measureSveltePerf = true;
```

Comment:
Removed this because it was always true. Let me know if there's a reason to preserve it.